rpc_jobs/rpc_maas.yml

```diff
@@ -0,0 +1,43 @@
+- project:
+    name: "rpc-maas-master-xenial-premerge"
+    repo_name: "rpc-maas"
+    repo_url: "https://github.com/rcbops/rpc-maas"
+    series: "master"
+    branches:
+      - "master.*"
+    image:
+      - xenial:
+          IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
+    scenario:
+      - master
+      - pike
+      - ocata
```

Comment:
In an effort to save some resources, I suggest adding ``TRIGGER_PR_PHRASE_ONLY: true`` to ocata, pike & master as they are not required PR's in the repo.

Reply:
I've done this for master, ocata and kilo. I think pike should be required given the current state of rpc-openstack so I'll make that so.